web-ui/src/main/resources/catalog/components/metadataactions/RelatedResourcesService.js

```diff
@@ -126,6 +126,8 @@
 
       var addWMSToMap =
         gnViewerSettings.resultviewFns && gnViewerSettings.resultviewFns.addMdLayerToMap;
+      var add3dTilesToMap =
```

Comment:
Does it work in GN map viewer ?

Reply:
i have no idea since i never really used/tried the internal GN viewer

Reply:
and to be fully honest, since i dont know the internals of the GN viewer at all, i added 3dtiles everywhere i saw wms/wmts until opening 3dtiles in external viewer worked, so maybe its not needed for my usecase.